YesterdayApi.UserIdentity.Data/YesterdayApi.UserIdentity.Data.csproj

```diff
@@ -1,7 +1,7 @@
 <Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <TargetFramework>netcoreapp2.1</TargetFramework>
+    <TargetFramework>netcoreapp2.2</TargetFramework>
   </PropertyGroup>
 
   <ItemGroup>
```

Comment:
You only updated target. Try to do the same with packages (like you did with YesterdayApiTest.csproj)
<PackageReference Include="Microsoft.AspNetCore.Identity.EntityFrameworkCore" Version="2.1.0" />

Reply:
The packages are working and up to date, i ran the project :)

Reply:
Great then! If packages are updated then it's fine 🙆‍♂️ Please notice coworkers that they NEED to update dotnet to 2.2 😀